geppetto-ui/src/components/html-viewer/showcase/HTMLViewerShowcase.less

```diff
@@ -0,0 +1,62 @@
+.htmlviewer > a {
+  background: transparent;
+}
+.htmlviewer > a:active,
+.htmlviewer > a:hover {
+  outline: 0;
+}
+a {
+  color: #428bca;
+  text-decoration: none;
+}
+.htmlviewer > a:hover,
+.htmlviewer > a:focus {
+  color: #2a6496;
+  text-decoration: underline;
+}
+.htmlviewer > a:focus {
+  outline: thin dotted;
+  outline: 5px auto -webkit-focus-ring-color;
+  outline-offset: -2px;
+}
+
+.htmlviewer > a.text-primary:hover {
+  color: #3071a9;
```

Comment:
@tarelli have to think about these kinds of styling. I'm wondering about banning all the styling from inside components that is not pure internal layout.

Reply:
@filippomc in geppetto we used to have some less variables primaryColour etc, we should approve say the material-ui structure, make a default one for geppetto and reference variables from it in all components.